html/global_attributes.json

```diff
@@ -846,6 +846,54 @@
           }
         }
       },
+      "enterkeyhint": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Global_attributes/enterkeyhint",
```

Comment:
```suggestion
```
IMHO if there’s not yet any MDN article for a feature, it’s better to omit the `mdn_url`